variables.tf

```diff
@@ -504,3 +504,9 @@ variable "rds_max_allowed_packet" {
   type        = number
   default     = 134217728
 }
+
+variable "disallow_incompatible_col_type_changes" {
+  description = "Hive metastore setting to disallow validation when incompatible schema type changes. Default is false."
+  type        = bool
+  default     = false
```

Comment:
default value should be `true` for backward compatibility, we can override this in internal deployment.

Reply:
So, in case that I set this to true I think it will disallow the setting, which means that it won't be validated
True -> No validation
False -> Validation

Is that what you want?

Reply:
I think it is opposite
true -> hive will validate schema changes
false -> hive won't validate schema changes

Reply:
Yes, I just checked and it's true, I just changed :)